test/unit/index.js

```diff
@@ -362,51 +362,51 @@ test('tracker with minimal token and one immediate block update with changes', a
   }
 })
 
-test('tracker with broken provider', async function (t) {
```

Comment:
Hey @mcmire, any thoughts on why this test keeps logging `JsonRpcProvider failed to detect network and cannot start up; retry in 1s (perhaps the URL is wrong or the node is not started)` every x seconds? The tests themselves are all passing, but the complete test suite never finishes.

Reply:
I believe this is related to https://github.com/MetaMask/eth-block-tracker/issues/301

Reply:
If this were due to the block tracker, we should have already seen this error when running the tests.

I think this might be new behavior in Ethers 6. There's a bunch of extra stuff in BrowserProvider and JsonRpcProvider that wasn't in Ethers 5. Sigh. This is making me think that we shouldn't actually switch to Ethers. I'll have to take a closer look at this.

Reply:
Sure thanks. I will rollback to this commit https://github.com/MetaMask/eth-token-tracker/pull/155/commits/08d7ed745c3fd9109109b465f8f04570f640bade